lib/msal-core/src/utils/WindowUtils.ts

```diff
@@ -341,13 +343,14 @@ export class WindowUtils {
     /**
      * Removes url fragment from browser url
      */
-    static clearUrlFragment(): void {
+    static clearUrlFragment(contentWindow: Window): void {
         // Office.js sets history.replaceState to null
         if (typeof history.replaceState === "function") {
             // Full removes "#" from url
-            history.replaceState(null, null, `${window.location.pathname}${window.location.search}`);
+            contentWindow.location.hash = "";
+            contentWindow.history.replaceState(null, null, `${contentWindow.location.pathname}${contentWindow.location.search}`);
         } else {
-            window.location.hash = "";
+            contentWindow.location.hash = "";
```

Comment:
It looks like this line can always be run (before the if), then only thing run in the if is `contentWindow.history.replaceState`.

Reply:
Right, thanks!